lesson01/main.tf

```diff
@@ -1,3 +1,11 @@
+terraform {
+  backend "s3" {
+    bucket = "wizeline-academy-terraform"
+    key    = "academy-user-N/terraform-academy/lesson01/terraform_lesson01.tfstate"
```

Comment:
Shouldn't this be passed as a command line parameter? So each student can specify their own without running the risk of everyone using the same.